helm/nessie/values.yaml

```diff
@@ -46,8 +46,12 @@ rocksdb:
 dynamodb:
   # -- The AWS region to use.
   region: us-west-2
+  # -- The name of the profile that should be used, when loading AWS credentials from a profile
+  # file. Required only if no secret is provided below.
+  profile: default
```

Comment:
Shouldn't the default be `nil`, since it's optional?

Reply:
I think that's fine, the profile won't be used anyways if the secret is present.

Reply:
Fair enough